### ISSUE TITLE
Documentation generation not working on windows

### ISSUE THREAD

**Issue by rritoch** [weavejester/codox#110]
The translations of native paths to URI is not functioning properly. The following exception was thrown when attempting to generate documentation on windows.  The same documentation generated without problems on linux.

```
D:\Users\Ralph\workspace\clj-nativedep>lein codox
Exception in thread "main" java.net.URISyntaxException: Illegal character in pat
h at index 3: doc\intro.html
        at java.net.URI$Parser.fail(URI.java:2848)
        at java.net.URI$Parser.checkChars(URI.java:3021)
        at java.net.URI$Parser.parseHierarchical(URI.java:3105)
        at java.net.URI$Parser.parse(URI.java:3063)
        at java.net.URI.<init>(URI.java:588)
        at hiccup.util$eval748$fn__749.invoke(util.clj:49)
        at hiccup.util$eval731$fn__732$G__722__737.invoke(util.clj:46)
        at hiccup.element$link_to.doInvoke(element.clj:15)
        at clojure.lang.RestFn.applyTo(RestFn.java:139)
        at clojure.core$apply.invoke(core.clj:617)
        at hiccup.def$wrap_attrs$fn__1058.doInvoke(def.clj:25)
        at clojure.lang.RestFn.invoke(RestFn.java:421)
        at codox.writer.html$topics_menu$iter__1306__1310$fn__1311.invoke(html.c
lj:167)
        at clojure.lang.LazySeq.sval(LazySeq.java:42)
        at clojure.lang.LazySeq.seq(LazySeq.java:60)
        at clojure.lang.RT.seq(RT.java:484)
        at clojure.core$seq.invoke(core.clj:133)
        at clojure.core$map$fn__4207.invoke(core.clj:2479)
        at clojure.lang.LazySeq.sval(LazySeq.java:42)
        at clojure.lang.LazySeq.seq(LazySeq.java:60)
        at clojure.lang.RT.seq(RT.java:484)
        at clojure.core$seq.invoke(core.clj:133)
        at clojure.core$apply.invoke(core.clj:617)
        at hiccup.compiler$eval885$fn__886.invoke(compiler.clj:88)
        at hiccup.compiler$eval854$fn__855$G__845__860.invoke(compiler.clj:68)
        at clojure.core$map$fn__4207.invoke(core.clj:2485)
        at clojure.lang.LazySeq.sval(LazySeq.java:42)
        at clojure.lang.LazySeq.seq(LazySeq.java:60)
        at clojure.lang.RT.seq(RT.java:484)
        at clojure.core$seq.invoke(core.clj:133)
        at clojure.core$apply.invoke(core.clj:617)
        at hiccup.compiler$eval885$fn__886.invoke(compiler.clj:88)
        at hiccup.compiler$eval854$fn__855$G__845__860.invoke(compiler.clj:68)
        at hiccup.compiler$render_element.invoke(compiler.clj:78)
        at hiccup.compiler$eval889$fn__890.invoke(compiler.clj:85)
        at hiccup.compiler$eval854$fn__855$G__845__860.invoke(compiler.clj:68)
        at clojure.core$map$fn__4207.invoke(core.clj:2487)
        at clojure.lang.LazySeq.sval(LazySeq.java:42)
        at clojure.lang.LazySeq.seq(LazySeq.java:60)
        at clojure.lang.Cons.next(Cons.java:39)
        at clojure.lang.RT.boundedLength(RT.java:1654)
        at clojure.lang.RestFn.applyTo(RestFn.java:130)
        at clojure.core$apply.invoke(core.clj:617)
        at hiccup.compiler$eval885$fn__886.invoke(compiler.clj:88)
        at hiccup.compiler$eval854$fn__855$G__845__860.invoke(compiler.clj:68)
        at clojure.core$map$fn__4207.invoke(core.clj:2485)
        at clojure.lang.LazySeq.sval(LazySeq.java:42)
        at clojure.lang.LazySeq.seq(LazySeq.java:60)
        at clojure.lang.RT.seq(RT.java:484)
        at clojure.core$seq.invoke(core.clj:133)
        at clojure.core$apply.invoke(core.clj:617)
        at hiccup.compiler$eval885$fn__886.invoke(compiler.clj:88)
        at hiccup.compiler$eval854$fn__855$G__845__860.invoke(compiler.clj:68)
        at hiccup.compiler$render_element.invoke(compiler.clj:78)
        at hiccup.compiler$eval889$fn__890.invoke(compiler.clj:85)
        at hiccup.compiler$eval854$fn__855$G__845__860.invoke(compiler.clj:68)
        at clojure.lang.Var.invoke(Var.java:415)
        at codox.writer.html$index_page.invoke(html.clj:232)
        at codox.writer.html$write_index.invoke(html.clj:341)
        at codox.writer.html$write_docs.invoke(html.clj:356)
        at clojure.lang.Var.invoke(Var.java:415)
        at codox.main$generate_docs.invoke(main.clj:94)
        at user$eval638.invoke(form-init6898987374814389404.clj:1)
        at clojure.lang.Compiler.eval(Compiler.java:6619)
        at clojure.lang.Compiler.eval(Compiler.java:6609)
        at clojure.lang.Compiler.load(Compiler.java:7064)
        at clojure.lang.Compiler.loadFile(Compiler.java:7020)
        at clojure.main$load_script.invoke(main.clj:294)
        at clojure.main$init_opt.invoke(main.clj:299)
        at clojure.main$initialize.invoke(main.clj:327)
        at clojure.main$null_opt.invoke(main.clj:362)
        at clojure.main$main.doInvoke(main.clj:440)
        at clojure.lang.RestFn.invoke(RestFn.java:421)
        at clojure.lang.Var.invoke(Var.java:419)
        at clojure.lang.AFn.applyToHelper(AFn.java:163)
        at clojure.lang.Var.applyTo(Var.java:532)
        at clojure.main.main(main.java:37)
Subprocess failed
```

**Comment by weavejester** [comment]
Sorry for the late response. Which version of Codox are you using?

**Comment by rritoch** [comment]
Hi, I am using version 0.9.1 

See: https://github.com/rritoch/clj-nativedep/blob/master/project.clj

**Comment by weavejester** [comment]
Looks like Codox isn't accounting for Windows path syntax in many of its links. I'll see about releasing a fix, but patches are welcome (as it's difficult for me to test on Windows).

**Comment by weavejester** [comment]
This *should* be fixed in 0.9.2. Can you check and see?

**Comment by rritoch** [comment]
Thank you for the repair. I was able to generate the documentation on windows 10 using the new 0.9.2 branch. You can see the committed the results at https://github.com/rritoch/clj-nativedep/commit/7d49ab64bd2681440e57595545122bad76655436
